rebar.config

```diff
@@ -1,6 +1,10 @@
+%% -*- mode:erlang -*-
 {minimum_otp_vsn, "21.0"}.
 
-{deps, [{jiffy, {git, "https://github.com/emqx/jiffy", {tag, "1.0.5"}}}]}.
+{deps, [ {jiffy, {git, "https://github.com/emqx/jiffy", {tag, "1.0.5"}}}
+       , {snabbkaffe, {git, "https://github.com/kafka4beam/snabbkaffe", {tag, "0.8.2"}}}
```

Comment:
test dependency ?

Reply:
ah. sorry, it actually instruments production code.

Reply:
The release build still needs the think layer over logger that release-mode snabbkaffe provides.